### PR TITLE
[ticket/15638] Add "word-break" to prevent PHP information of ACP from overflowing

### PR DESCRIPTION
**Pull request by ioannisbat**
PHPBB3-15638

Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [ ] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-15638

### CODE REVIEW

File: phpBB/adm/style/admin.css [hunk 1]
```diff
@@ -840,6 +840,7 @@ table.zebra-table tbody tr:nth-child(odd) {
 }
 
 .row2 {
+	word-break: break-all;
```
Comment (Crizz0) [inline phpBB/adm/style/admin.css]: Why not after the second property to keep in alphabetical classification?

**Comment by Crizz0** [comment]
Hi,

You don't need a new pull request. All you need is a rebase, which fixes the commit message: https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History 

Commit-message format: https://area51.phpbb.com/docs/dev/3.2.x/development/git.html#commit-messages 

Regards

File: phpBB/adm/style/admin.css (hunk 1 above)
Comment (hanakin) [reply phpBB/adm/style/admin.css]: @Crizz0 We do no use alphabetical order for css we use font, design, layout order

Comment (Crizz0) [reply phpBB/adm/style/admin.css]: ah, okay.

**Comment by ioannisbat** [comment]
@hanakin There is an "Unexpected EOF encountered" error in my latest commit on Travis and I can't understand the reason. Do you have any idea what's the problem?

**Comment by ioannisbat** [comment]
@DavidIQ Can you elaborate more on the 3rd one. I can't understand which is the problem

**Comment by DavidIQ** [comment]
Well you've closed this so not sure what your intentions are at this point. In any case the error is likely that you have either tabs in your commit message or have an extra line at the end of your commit message where there shouldn't be.

**Pull request closed without merging**